app/presenters/tree_builder_ops.rb

```diff
@@ -21,7 +21,7 @@ def x_get_tree_zone_kids(object, count_only)
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
     region = MiqRegion.my_region
-    objects = region.zones.sort_by { |z| z.name.downcase }
+    objects = region.zones.visible.sort_by { |z| z.name.downcase }
```

Comment:
I don't think we should hide zones in the settings...otherwise the user can never get to EMSes in the invisible zones.

Reply:
@Fryguy When user clicks on "suspend provider" he doesn't know if there is some maintenance zone, he wants his provider paused and out of automatic processing. I'll be confusing if he come to settings and there is EMS in some weird zone with name containing UUID. He can try to fix it and delete (or edit) these zones, which is wrong. 
User shouldn't have possibility to do something with it this way (and delete maintenance zone, because it was refused to make auto creating on access). 

I understand your point, can it be done in other way if necessary? 
- In settings/toolbar - buttons Edit zone and Delete zone are disabled.
- Color of text should be gray
- ...Or make separate branch in tree with both above

Reply:
This is not true, the settings doesn't expose the providers and we don't want to allow the user to edit the maintenance zone.